colorbleed/fusion/lib.py

```diff
@@ -38,3 +39,26 @@ def update_frame_range(start, end, comp=None, set_render_range=True):
 
     with avalon.fusion.comp_lock_and_undo_chunk(comp):
         comp.SetAttrs(attrs)
+
+
+def get_additional_data(container):
+    """Get Fusion related data for the container
+
+    Args:
+        container(dict): the container found by the ls() function
+
+    Returns:
+        dict
+    """
+
+    def clamp(value):
+        return int(value * 255)
+
+    tool = container["_tool"]
+    tile_color = tool.TileColor
+    if tile_color is None:
+        return {}
+
+    return {"color": QtGui.QColor(clamp(tile_color["R"]),
```

Comment:
Can't you avoid the `clamp` function and just use [QColor.fromRGBf](http://doc.qt.io/archives/qt-4.8/qcolor.html#fromRgbF)?